api/database/factories/ModelFactory.php

```diff
@@ -85,24 +85,15 @@
 
 $factory->define(App\Models\AuthToken::class, function ($faker) {
 
-    $hostname = env('APP_HOST', 'localhost');
-
     $user = factory(App\Models\User::class)->make();
-    $now = new Carbon();
-
-    $body = [
-        'iss' => $hostname,
-        'aud' => str_replace('.api', '', $hostname),
-        'sub' => $user->user_id,
-        'nbf' => $now->timestamp,
-        'iat' => $now->timestamp,
-        'exp' => $now->addHour(1)->timestamp,
-        'jti' => $faker->regexify('[A-Za-z0-9]{8}'),
-        'user' => $user->toArray()
-    ];
 
     $jwtAuth = \App::make('Tymon\JWTAuth\JWTAuth');
     $token = $jwtAuth->fromUser($user);
 
-    return compact('token') + $body;
+    // Transform the user to be injected in the token for the api doc.
+    $transformerService = App::make(App\Services\TransformerService::class);
+    $transformer = new App\Http\Transformers\IlluminateModelTransformer($transformerService);
+    $user = $transformer->transform($user);
+
+    return ['token' => $token, 'user' => $user];
```

Comment:
If this transformation is only used by the documentation, then the response structure should just be constructed in the documentation eg.

```
{!!
                [
                    token: $factory->make(\App\Models\AuthToken::class)->token,
                    decodedToken:  ???
                ]
            !!}
```